first-run

```diff
@@ -5,4 +5,9 @@
 
 if [ -n "$PASSWORD" -a -z "$SKIP_INSTALL" ]; then
     unattendedinstall
+fi
+
+
+if [ -z "$DEBUG_LOGGING" ]; then
+  sed -i 's/AsyncLogger name="se.curity" level="DEBUG"/AsyncLogger name="se.curity" level="INFO"/g' "$IDSVR_HOME/etc/log4j2.xml"
```

Comment:
do the same for `io.curity` logger as well.

Reply:
did the opposite for that logger, since it is INFO by default

Reply:
Perfect

Reply:
@travisspencer we have some other loggers to DEBUG by default, should we control them with the same flag? 
i.e: se.curity.identityserver.config.ConfDListener, se.curity.identityserver.config.ConfDConnection, se.curity.identityserver.config

If so, we can globally change DEBUG for INFO and visa versa in the log4j2.xml file